packages/auth/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN --mount=type=cache,id=pnpm,target=/pnpm/store \
 FROM base AS prod-deps
 
 COPY package.json pnpm-workspace.yaml .npmrc ./
+COPY packages/auth/knexfile.js ./packages/auth/knexfile.js
```

Comment:
this line here is not really needed (same for `packages/backend/Dockerfile` however, I like consistency (line #55)